port/unix/omrsock.c

```diff
@@ -362,7 +362,15 @@ omrsock_freeaddrinfo(struct OMRPortLibrary *portLibrary, omrsock_addrinfo_t hand
 int32_t
 omrsock_sockaddr_init(struct OMRPortLibrary *portLibrary, int32_t family, uint8_t *addrNetworkOrder, uint16_t portNetworkOrder, omrsock_sockaddr_t handle)
 {
-	return OMRPORT_ERROR_NOT_SUPPORTED_ON_THIS_PLATFORM;
+	omr_os_sockaddr_in *sockAddr = NULL;
+
+	sockAddr = (omr_os_sockaddr_in *) &handle->data;
```

Comment:
You can combine these two lines into one

Reply:
I don't understand how all this works. `handle` points to some `data`, but then we clear the handle.

Reply:
Basically, sockAddr is supposed to point to the data element of handle. Later on, we set sockAddr->sin_family, sockAddr->sin_port and so on to what it was supposed to be.  We c;ear handle because handle contains the data element, so we are essentially setting data to 0 before setting data.sin_family, data.sin_port and so on.